free-3.md

```diff
@@ -10,59 +10,116 @@ _What need is met by your idea? Who are you helping? What is that person's
+the input program) to see if a new process needed to be created for
+the given job.
+
+Errors might involve failure to start at a specified time, due to
+not having processor time at that moment, or any number of file system or
+operating system failures.  In any case, the error should probably be
+best logged to a file *and* reported to the user via a shell notification.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to create periodic process and to create a monitoring
+process.  It should be more difficult to create jobs that are too interactive
+since it could be difficult if multiple were triggered at the same
+time, so we might consider restricting to background processes.  It should
+be impossible for user error (or creator's error) to create too many processes
+which would overwhelm the processor or take up too many resources.
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+Yes, it seems that many people already beat me to it:
+ * [timely](https://github.com/Factual/timely)
+ * [scheduling](https://dzone.com/articles/building-dsl-scheduling-tasks)
+They seem to address the need quite effectively, which makes me think that my
+DSL project seem as thought it might be superfluous.
+
+I like how the DZone project focuses extensively on fluency and composability
+of the DSL designed.  It helped give me ideas for how to represent the
+conditions where a task starts when a condition is met, and clarifies whether
+or not this repeats or is a single trigger once the condition is satisfied.
 
 ## The Project
 This section examines whether the idea makes for a good CS 111 project.
 
-
 ### Suitability
 _If someone were to work on this project, what percentage of their time would be
 spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that less than half of the time working on this project would be
+involved in language design, since the implementation details of the project
+might be a bit gnarly.  I would expect that the query and spin style of the
+language might be a particularly difficult part to handle, especially if
+it involves interacting with different operating systems.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This seems like it would be quite a significant DSL and a serious time
+commitment, but largeley due to the non-design portions of the implementaiton.
```

Comment:
If you are just converting calls to cron I think this part could be very easy to implement.  Perhaps the added functionality would be rather difficult though...